File: Models/MOVIMIENTO.cs
namespace Models
{
    public class MOVIMIENTO
    {
        public int IdMovimiento { get; set; }
        public int IdTipoMovimiento { get; set; }
        public int CantDolar { get; set; }
        public decimal PUnitario { get; set; }
        public decimal CostoToTal { get; set; }
        public decimal Pago { get; set; }
        public decimal Cambio { get; set; }
        public string FecRegistro { get; set; }
        public int IdUsuario { get; set; }
    }
}

File: PIA_PD/ClientApp/src/logic/Movimiento.js
import axios from 'axios';

const ENDPOINT_PATH = 'api/movimiento'
export default {
    listaMovimientos() {
        return axios.get(ENDPOINT_PATH + '/Read');
    },

    registrarMovimiento(objeto){
        return axios.post(ENDPOINT_PATH+'/Create', {
            IdTipoMovimiento : objeto.nombre,
            CantDolares : objeto.aPaterno,
            CostoTotal : objeto.costoTotal,
            Pago : objeto.pago,
            Cambio : objeto.cambio,
            IdUsuario : objeto.idUsuario
        });
    }
}